src/paperoni/__main__.py

```diff
@@ -144,11 +144,13 @@ class Get(Productor):
 
         def run(self, work):
             focuses = deserialize(Focuses, work.focuses)
-            ex = work.exclusions and deserialize(set[str], work.exclusions)
+            ex = work.exclusions and deserialize(set[str], work.exclusions) or set()
             if work.state.exists():
                 top = deserialize(Top[Scored[PaperWorkingSet]], work.state)
             else:
                 top = Top(work.n)
+            # Avoid duplicata in the working set
+            ex.update(pinfo.key for scored in top for pinfo in scored.value.collected)
```

Comment:
This seams like a rather naive way of avoiding duplicata. Did you had other situations in mind where duplicata could occur?

Reply:
Identical links (e.g. both papers have arxiv:12345 in pinfo.paper.links). Identical titles (I suppose there could be false positives, but I don't think they are likely).